module/importer-ergonode-1/src/Infrastructure/Processor/ErgonodeImportProcess.php

```diff
@@ -61,6 +62,7 @@ public function start(Import $import): void
             }
         } catch (Throwable $exception) {
             $this->logger->critical($exception);
+            throw new ImportException('Can\'t process file');
```

Comment:
First of all you shoould catch exception `\Ergonode\ImporterErgonode1\Infrastructure\Reader\Exception\ErgonodeZipExtractorException` -- this will claim invalid zip source in general. 

Second of all appropriate exception probably should be thrown in 
\Ergonode\ImporterErgonode1\Infrastructure\Reader\AbstractErgonodeReader claiming which file is missing and mapped to ImportException